pkg/services/featuremgmt/toggles_gen.json

```diff
@@ -1219,14 +1219,18 @@
     {
       "metadata": {
         "name": "groupToNestedTableTransformation",
-        "resourceVersion": "1718727528075",
-        "creationTimestamp": "2024-02-07T14:28:26Z"
+        "resourceVersion": "1720568606781",
+        "creationTimestamp": "2024-02-07T14:28:26Z",
+        "annotations": {
+          "grafana.app/updatedTimestamp": "2024-07-09 23:43:26.781894 +0000 UTC"
+        }
       },
       "spec": {
         "description": "Enables the group to nested table transformation",
-        "stage": "preview",
+        "stage": "GA",
         "codeowner": "@grafana/dataviz-squad",
-        "frontend": true
+        "frontend": true,
+        "expression": "true"
```

Comment:
Everything looks good, but what is this for? Why was it not there before?

Reply:
This sets the feature toggle as default true in Grafana OSS (whereas [we have a separate process for managing feature toggle rollout in Grafana Cloud](https://github.com/grafana/deployment_tools/pull/155211))

So in v11.2 this transformation will show up by default in Grafana OSS - right now it has to be manually enabled